setup.py

```diff
@@ -55,7 +55,6 @@ def find_version(*file_paths):
         'flask>=0.10.0',
         'lxml>=3.4.4',
         'spyne>=2.12.8',
-        'nose',
```

Comment:
> Ideally we should read in from the requirements file here too to prevent the dependencies from keeping out of sync.

Yes, this isn't ideal, but ok for now?

> Once odm2api is updated on PyPI I will make the change.

There hasn't been movement on this. I suspect a new `PyPI` package could be created soon based on an `odm2api` release that freezes the current master. I'm told a big `odm2api` update is coming, so it probably would be a good idea to do such a release.

Thoughts?